src/verify_cert.rs

```diff
@@ -141,7 +172,7 @@ fn check_signatures(
     supported_sig_algs: &[&SignatureAlgorithm],
     cert_chain: &Cert,
     trust_anchor: &TrustAnchor,
-    crls: &[&dyn CertRevocationList],
+    revocation: &Option<RevocationOptions>,
```

Comment:
I don't think `RevocationOptions` is `Copy` because of the `&'a [&'a dyn CertRevocationList]` field. We don't want to move it so passing by ref seemed like the right thing to do.

Reply:
IIRC references are always `Copy`.

Reply:
Ah you're totally right. My mistake. Deriving `Copy` and dropping the `&` to pass by value works fine.

Reply:
Fixed up to pass by value.